examples/vision/autoencoder.py

```diff
@@ -125,8 +116,8 @@ def display(dataset1, dataset2):
 autoencoder.summary()
 
 """
-Now we can train the autoencoder using `train_dataset` as our input `X` and the same
-dataset as our `y`. Notice that we are setting up the validation data using the same format.
+Now we can train the autoencoder using `train_dataset` as our input input data and the same
```

Comment:
nit: repetition ("input" x 2)

```suggestion
Now we can train the autoencoder using `train_dataset` as our input data and the same
```